src/ShiftRegister74HC595.cpp

```diff
@@ -71,9 +83,15 @@ void ShiftRegister74HC595::set(int pin, uint8_t value)
 }
 
 // Updates the shift register pins to the stored output values.
+// This is the function that actually writes data into the shift registers of the 74HC595
 void ShiftRegister74HC595::updateRegisters()
 {
-	 setAll(_digitalValues);
+    for (int i = _numberOfShiftRegisters - 1; i >= 0; i--) {
+        shiftOut(_serialDataPin, _clockPin, MSBFIRST, digitalValues[i]);
```

Comment:
Little typo, should be `_digitalValues`, here.

Reply:
I have taken care of that one in this commit: https://github.com/Simsso/ShiftRegister74HC595/pull/15/commits/c496675fee4e48a53c5de7297f8fd9d09e0ca41b